src/gaggiuino.ino

```diff
@@ -126,6 +119,10 @@ void setup(void) {
   scalesInit(eepromCurrentValues.scalesF1, eepromCurrentValues.scalesF2);
   LOG_INFO("Scales init");
 
+  // Pump init
+  pumpInit(eepromCurrentValues.powerLineFrequency);
```

Comment:
PSM::cps() should return this, ideally we should deprecate the configured version.

Reply:
I fully agree, this is a bigger step than anticipated and needs to be addressed separately.